src/FlaUI.Core/AutomationElements/TextBox.cs

```diff
@@ -48,10 +73,70 @@ public string Text
                 }
                 else
                 {
-                    Enter(value);
+                    if (SetTextWin32(value) == false)
+                    {
+                        Enter(value);
+                    }
                 }
             }
         }
+        
+        private bool SetTextWin32(string text)
+        {
+		// try with native Win32 function SetWindowText
+		if (Properties.NativeWindowHandle.IsSupported)
+		{
+			var windowHandle = Properties.NativeWindowHandle.ValueOrDefault;
+			if (windowHandle != IntPtr.Zero)
+			{
+				IntPtr textPtr = IntPtr.Zero;
+				try
+				{
+					textPtr = Marshal.StringToBSTR(text);
+				}
+				catch { }
+
+				try
```

Comment:
Is this try needed? What could happen here?

Reply:
I removed the try.